sample-code/ic-customized-workflow-app/IC_customized_workflow_using_S3.sh

```diff
@@ -32,7 +32,7 @@ final_output_url='A presigned PUT URL for final result'
 
 
 #--------------------------------------------- Call Sensei CutOut Service -----------------------------------------
```

Comment:
change this line to `Call Remove Background Service`?

Reply:
Done